yuca/ca/continuous.py

```diff
@@ -768,119 +546,6 @@ def get_frame(self, universe, mode=0):
         elif mode == 4:
             return new_universe, universe, neighborhoods, update 
 
-    def fit_growth(self, target, **kwargs):
```

Comment:
`fit_growth` is an old experimental feature that was deprecated and now removed.